extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.js

```diff
@@ -633,6 +633,13 @@ export class AmpLightboxGallery extends AMP.BaseElement {
    * @param {!SwipeDef} data
    */
   swipeGesture_(data) {
+    // Check if the carousel has been cleared out due to close. It is unclear
```

Comment:
The carousel is created on the first tap (I believe), so could it be a race condition where someone taps lightbox for the first time, and then triggers a swipe before the carousel has been initialized?

Reply:
My thought is that the gestures are coming in out of order. If `data.first` doesn't come first, we'll have a null element reference.

Reply:
I'm not sure, since the gesture is actually set up on the carousel element itself.

Reply:
> My thought is that the gestures are coming in out of order. If `data.first` doesn't come first, we'll have a null element reference.

Yeah that is possible, would need to investigate that possibility,

Reply:
I thought it was the swipe element that was null, not the carousel? The swipe element gets initialized in the `startSwipe`, which happens when `data.first` is true.